src/components/Footer/Footer.js

```diff
@@ -1,115 +1,51 @@
+import MermaidDocsBIN from '../../docs/MERMAID-user-docs-B-IN-min.pdf'
 import OfflineHide from '../generic/OfflineHide'
 import OfflineToggle from '../OfflineToggle'
-import theme from '../../theme'
-
-const smallFooterTextStyle = css`
-  font-size: ${theme.typography.smallFontSize};
-  ${mediaQueryPhoneOnly(css`
-    margin: ${theme.spacing.small} 0;
-  `)}
-`
 
-const VersionWrapper = styled('span')`
-  ${smallFooterTextStyle}
-`
+const Footer = () => {
+  const { isAppOnline } = useOnlineStatus()
+  const [isDropdownOpen, setIsDropdownOpen] = useState(false)
+  const dropdownRef = useRef(null)
 
-const StyledFooter = styled('footer')`
-  text-align: end;
-  background-color: ${theme.color.footerColor};
-  padding: ${theme.spacing.small};
-  ${mediaQueryPhoneOnly(css`
-    text-align: start;
-    padding-bottom: 5rem;
-  `)}
-  p,
-  label,
-  nav a,
-  button {
-    ${smallFooterTextStyle}
-  }
-`
-const offlineToggleSize = theme.typography.defaultIconSize
-const CssToggle = styled('span')`
-  display: block;
-  width: calc(${offlineToggleSize} * 2);
-  position: relative;
-  &:before,
-  &:after {
-    content: '';
-    transition: 0.3s;
-    position: absolute;
-  }
-  &:before {
-    // container
-    width: calc(${offlineToggleSize} * 2);
-    height: ${offlineToggleSize};
-    left: 0;
-    top: 0;
-    border-radius: 25% / 50%;
-    background: green;
-  }
-  &:after {
-    // toggle
-    width: calc(${offlineToggleSize} - 2px);
-    height: calc(${offlineToggleSize} - 2px);
-    top: 1px;
-    left: 1px;
-    background: oldlace;
-    border-radius: 50%;
-  }
-`
-const StyledToggleLabel = styled('label')`
-  background: rgba(255, 255, 255, 0.5);
-  position: fixed;
-  bottom: 0;
-  left: 0;
-  cursor: pointer;
-  display: flex;
-  gap: 1.5rem;
-  padding: ${theme.spacing.small};
-  margin: 0;
-  input {
-    display: none;
-  }
-  input:checked {
-    ~ span:after {
-      background: ${theme.color.cautionColor};
-      left: calc(${offlineToggleSize} - 1px);
-    }
-    ~ span:before {
-      background: ${theme.color.cautionColor};
-      opacity: 0.5;
-    }
-  }
-  input:disabled {
-    ~ span:before {
-      background: ${theme.color.disabledColor};
-      opacity: 0.5;
-      cursor: not-allowed;
+  useEffect(() => {
+    const handleClickOutsideDropdown = (event) => {
+      if (isDropdownOpen && dropdownRef.current && !dropdownRef.current.contains(event.target)) {
+        setIsDropdownOpen(false)
+      }
     }
-    ~ span:after {
-      background: ${theme.color.disabledColor};
-      cursor: not-allowed;
+
+    document.addEventListener('click', handleClickOutsideDropdown)
+
+    return () => {
+      document.removeEventListener('click', handleClickOutsideDropdown)
     }
-  }
-`
+  }, [isDropdownOpen])
 
-const FooterNav = styled('nav')`
-  a {
-    display: inline-block;
-    padding: 0 ${theme.spacing.small};
-  }
-`
+  const handleLanguageSelect = (event) => {
```

Comment:
Why use JS instead of just links with srcs set appropriately?

Reply:
used JS here since it's a button click on the option